GoogleDataTransport/GDTCORLibrary/GDTCORPlatform.m

```diff
@@ -251,6 +259,24 @@ + (void)load {
   [self sharedApplication];
 }
 
+/** Initializes sSemaphoreQueue and sBackgroundIdentifierToSemaphoreMap. */
++ (void)initialize {
+#if TARGET_OS_WATCH
+  if (self == [GDTCORApplication class]) {
```

Comment:
Why the class equality check? If a subclass of GDTCORApplication were to be made, this wouldn't work anymore.

Reply:
Removed class check.
- It depends on whether we want a subclass of GDTCORApplication to call the super initialize method. In our case, the queue and map will only initialize once for one application lifetime with dispatch once. So I think we can safely remove the class check.